src/hooks/useTransferToL1.js

```diff
@@ -103,6 +116,172 @@ export const useTransferToL1 = () => {
   );
 };
 
+export const useFastTransferToL1 = () => {
```

Comment:
What will happen if frontend is reloaded after fast transfer is initiated? Will it resume the withdrawal process?